lightning/src/ln/outbound_payment.rs

```diff
@@ -1917,7 +1925,7 @@ impl_writeable_tlv_based_enum_upgradable!(PendingOutboundPayment,
 		(2, retry_strategy, required),
 		(4, max_total_routing_fee_msat, option),
 		(5, invoice_request, option),
-		(6, context, required),
+		(6, nonce, required),
```

Comment:
We may need to make this an `Option` otherwise we won't be able to support downgrading. Might be fine though except for long-lived `Refund`s (cc: @TheBlueMatt).